### PR TITLE
TRD updates

### PR DESCRIPTION
**Pull request by martenole**
- small fix for the tracking QC which needs to get the GRP now from CCDB
- Trap2CRU now sends different trigger depending on whether digits were suppressed (physics) or included (calib)

### CODE REVIEW

File: Detectors/TRD/simulation/src/Trap2CRU.cxx [hunk 1]
```diff
@@ -613,32 +611,35 @@ void Trap2CRU::convertTrapData(o2::trd::TriggerRecord const& triggerrecord, cons
...
-          int firstDigitMCM = mCurrentDigit;
-          while (mDigits[mDigitsIndex[mCurrentDigit]].getMCM() == currentMCM &&
-                 mDigits[mDigitsIndex[mCurrentDigit]].getROB() == currentROB &&
-                 mDigits[mDigitsIndex[mCurrentDigit]].getHCId() == hcid) {
-            mCurrentDigit++;
-            digitcounter++;
-            if (mCurrentDigit == enddigitindex) {
-              break;
+
+        if (isCalibTrigger) {
```
Comment (martenole) [inline Detectors/TRD/simulation/src/Trap2CRU.cxx:615]: @bazinski if I send a physics trigger and still write the digit HC headers and raw end markers then the raw reader gets quite confused:
```
[40125:trd-datareader]: [20:41:02][INFO] 2 current endpont : 1 previous end point : 0
[40125:trd-datareader]: [20:41:02][INFO] 1 current endpont : 1 previous end point : 0
[40125:trd-datareader]: [20:41:02][INFO] 2 current endpont : 1 previous end point : 0
[40125:trd-datareader]: [20:41:02][INFO] 2 current endpont : 1 previous end point : 0
[40125:trd-datareader]: [20:41:02][INFO] 2 current stopbit: 5 previous stopbit: 1
[40125:trd-datareader]: [20:41:02][INFO] 1 current stopbit: 5 previous stopbit: 1
[40125:trd-datareader]: [20:41:02][INFO] 2 current stopbit: 5 previous stopbit: 1
[40125:trd-datareader]: [20:41:02][INFO] 2 current stopbit: 5 previous stopbit: 1
```
But the way it is implemented here it works fine

**Comment by martenole** [comment]
Here I also changed that no more triggers without any digits or tracklets are written out. So in synthetics we should not see the "bogus trigger" messages for the TRD anymore. But I still need to test it with a larger 500kHz pp TF to check that also the BC and orbit are not nonsense.

File: Detectors/TRD/simulation/src/Trap2CRU.cxx (hunk 1 above)
Comment (bazinski) [reply Detectors/TRD/simulation/src/Trap2CRU.cxx:615]: It should just reject the data after the tracklet end marker, and then go in search of the next tracklethcheader. you might end up with more warnings/info messages than you would care to deal with.

Comment (bazinski) [reply Detectors/TRD/simulation/src/Trap2CRU.cxx:615]: The raw data off the detector does not have digit half chamber headers for physics triggers.

**Review by bazinski — DISMISSED** [review]

**Comment by martenole** [comment]
Failure is unrelated. This is ready to be merged

**Pull request merged**